File: 프로그래머스/unrated/181864. 문자열 바꿔서 찾기/문자열 바꿔서 찾기.cpp
#include <string>
#include <vector>

using namespace std;

int solution(string myString, string pat) {
    for(auto &str : myString){
        if(str == 'A')
            str = 'B';
        else
            str = 'A';
    }
    if(myString.find(pat) == string::npos) return 0;
    return 1;
}

File: 프로그래머스/unrated/181864. 문자열 바꿔서 찾기/문자열 바꿔서 찾기.java
class Solution {
    public int solution(String myString, String pat) {
        String rev = "";
        for(int i=0;i<myString.length();i++){
            if(myString.charAt(i) == 'A'){
                rev += "B";
            }
            else{
                rev += "A";
            }
        }
        System.out.println(rev);
        if(rev.indexOf(pat) == -1) return 0;
        return 1;
    }
}